src/pages/ExamsPage/components/ReviewExamAttemptButton.jsx

```diff
@@ -96,7 +96,7 @@ const ReviewExamAttemptButton = ({
           </ul>
           <p>{formatMessage(messages.ReviewExamAttemptModalBodySessionInfo)}</p>
           <ActionRow>
-            <Button as="a" href={getLaunchUrlByExamId(attemptId)} variant="link">
+            <Button as="a" href={getLaunchUrlByExamId(attemptId)} target="_blank" variant="link">
```

Comment:
I just noticed my own bug here. This takes exam id but we're passing in the attempt id.

Reply:
Ah cool, I'll update that

Reply:
@Zacharis278, just taking a look at this. Is it better to add exam as a prop (which probably means it would need to be changed elsewhere too) or to use a getter (if it exists) to get the exam by exam name or something like that? I see that exam name and attempt id are props for the review button but there is no exam prop.

Reply:
darn too bad it isn't just available already. But yeah it should get passed down as a property